src/main/java/org/spongepowered/api/world/World.java

```diff
@@ -334,28 +332,6 @@ default Difficulty getDifficulty() {
         return getProperties().getDifficulty();
     }
 
```

Comment:
Can you make the world also a GameRuleHolder? And then delegate all the methods to the properties.

Reply:
Yeah, I will just specify in the Javadoc that most implementations delegate the methods to the properties.